### ISSUE TITLE
Search breaks for some search terms

### ISSUE THREAD

**Issue by straight-shoota** [crystal-lang/crystal-book#127]
Search usually works fine but for some search terms it fails and there are no results nor error messages on the page. For example, search for [`char`](https://crystal-lang.org/docs/?q=char)
The JavaScript console shows this error:
```
lunr.min.js:7 Uncaught TypeError: Cannot read property 'tf' of undefined
    at t.Index.documentVector (lunr.min.js:7)
    at t.Index.<anonymous> (lunr.min.js:7)
    at Array.map (<anonymous>)
    at t.SortedSet.map (lunr.min.js:7)
    at t.Index.search (lunr.min.js:7)
    at LunrSearchEngine.search (search-lunr.js:34)
    at Object.query (search-engine.js:31)
    at launchSearch (search.js:79)
    at handleUpdate (search.js:121)
    at HTMLInputElement.<anonymous> (search.js:142)
```

According to olivernn/lunr.js#243 this might be caused by lunr.js having issues with unicode codepoints like emojis. There is an emoji (🔮) in the book on both pages about char and string literals and it fails to search for any terms that appear on one of these pages. I am not 100% sure about it, but this looks like strong evidence.

This failure should be fixed in lunar.js but until then, we should try to find a workaround for Crystal Book.
I think we could just remove the emojis from the pages. It's nice to have them there but they're not essential. Perhaps we could also manually remove them from the search index.

**Comment by straight-shoota** [comment]
It's possible to encode the emoji with `&#x1F52E;`, but that will be escaped in a fenced code block.
But it works in plain text and enables search.

**Comment by straight-shoota** [comment]
Fixed by #433
